web/pages/embed/video/index.tsx

```diff
@@ -40,10 +40,13 @@ export default function VideoEmbed() {
    * query parameters ourselves
    */
   const path = router.asPath.split('?')[1] ?? '';
-  const query = path.split('&').reduce((currQuery, part) => {
-    const [key, value] = part.split('=');
-    return { ...currQuery, [key]: value };
-  }, {} as Record<string, string>);
+  const query = path.split('&').reduce(
```

Comment:
why is this bot re-prettifying code that's already been prettified? I didn't even touch these lines of code.

Reply:
Maybe prettier updated rules, or maybe for some reason it didn't get prettified previously. It's annoying, but I wouldn't worry about it.